### ISSUE TITLE
adapter logs useless debug statements which make the logs harder to read.

### ISSUE THREAD

**Issue by jmrodri** [automationbroker/bundle-lib#158]
While running the broker recently with debug logging enabled the adapters are logging multiple times:
```
"Registry::responseToSpec"
"Registry::configToSpec"
```
Here is a snippet of a recent run.

```
time="2018-08-02T17:29:57Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:57Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:57Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/rhpam-apb:latest into Spec"
time="2018-08-02T17:29:57Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/rhpam-apb:latest runtime is 2"
time="2018-08-02T17:29:57Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:57Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:57Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/keycloak-apb:latest into Spec"
time="2018-08-02T17:29:57Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/keycloak-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/rhel-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/rhel-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/mssql-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/mssql-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/openshift-logging-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/openshift-logging-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/nginx-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/nginx-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:58Z" level=info msg="Didn't find encoded Spec label. Assuming image is not APB and skipping"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/photo-album-demo-api-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/photo-album-demo-api-apb:latest runtime is 2"
time="2018-08-02T17:29:59Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:59Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:59Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/galera-apb:latest into Spec"
time="2018-08-02T17:29:59Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/galera-apb:latest runtime is 2"
time="2018-08-02T17:29:59Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:59Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:59Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/manageiq-apb:latest into Spec"
time="2018-08-02T17:29:59Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/manageiq-apb:latest runtime is 2"
time="2018-08-02T17:29:59Z" level=debug msg="Registry::responseToSpec"
time="2018-08-02T17:29:59Z" level=debug msg="Registry::configToSpec"
time="2018-08-02T17:29:59Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/postgresql-apb:latest into Spec"
```

**Comment by jmrodri** [comment]
Proposed fix
-----
* remove the 2 debug statements

Proposed output
-------
```
time="2018-08-02T17:29:57Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/rhpam-apb:latest into Spec"
time="2018-08-02T17:29:57Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/rhpam-apb:latest runtime is 2"
time="2018-08-02T17:29:57Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/keycloak-apb:latest into Spec"
time="2018-08-02T17:29:57Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/keycloak-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/rhel-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/rhel-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/mssql-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/mssql-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/openshift-logging-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/openshift-logging-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/nginx-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/nginx-apb:latest runtime is 2"
time="2018-08-02T17:29:58Z" level=info msg="Didn't find encoded Spec label. Assuming image is not APB and skipping"
time="2018-08-02T17:29:58Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/photo-album-demo-api-apb:latest into Spec"
time="2018-08-02T17:29:58Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/photo-album-demo-api-apb:latest runtime is 2"
time="2018-08-02T17:29:59Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/galera-apb:latest into Spec"
time="2018-08-02T17:29:59Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/galera-apb:latest runtime is 2"
time="2018-08-02T17:29:59Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/manageiq-apb:latest into Spec"
time="2018-08-02T17:29:59Z" level=info msg="adapter::configToSpec -> Image docker.io/ansibleplaybookbundle/manageiq-apb:latest runtime is 2"
time="2018-08-02T17:29:59Z" level=debug msg="Successfully converted Image docker.io/ansibleplaybookbundle/postgresql-apb:latest into Spec"
```

**Comment by jmrodri** [comment]
Fixed by PR #159
